### ISSUE TITLE
Check when extra in log override default fields

### ISSUE THREAD

**Issue by orsinium** [wemake-services/wemake-python-styleguide#388]
# Rule request

## Thesis

When passed to logger extra parameters overwrite default logging fields like `name`, `lineno` or `message`, logging raises KeyError.

Example:

```python
import logging
logging.warning('msg', extra=dict(name='test'))
```

This raises next exception:
```python
KeyError: "Attempt to overwrite 'name' in LogRecord"
```

## Reasoning

In this case logging raises `KeyError` exception at runtime. It will be very useful to find out this at static checks stage.

**Comment by sobolevn** [comment]
@orsinium can you please send this request to `flake8-logger-format`?
We are relying on them for this logic.
